test/integration/rules/summary-name/summary-name.html

```diff
@@ -60,6 +60,11 @@
   Conflict resolution gets this to be ignored
 </details>
 
+<details>
+  <summary id="button-role-fail" role="button"></summary>
+  Conflict resolution gets this to be ignored
```

Comment:
Copy paste typo as there is no conflict resolution (same for the `role=heading` above).

```suggestion
  Hello world
```